tests/sentry/integrations/slack/service/test_slack_service.py

```diff
@@ -317,6 +326,112 @@ def test_no_parent_notification(self, mock_send):
         self.service.notify_all_threads_for_activity(activity=self.activity)
         assert not mock_send.called
 
+    def test_none_user_id_uptime_resolved(self):
+        """Test that uptime resolved notifications are allowed even without a user_id"""
+        self.group.update(type=UptimeDomainCheckFailure.type_id)
+        self.activity.update(
+            user_id=None,
+            type=ActivityType.SET_RESOLVED.value,
+        )
+
+        with mock.patch.object(self.service, "_notify_all_threads_for_activity") as mock_notify:
+            self.service.notify_all_threads_for_activity(activity=self.activity)
+            mock_notify.assert_called_once()
+
+    @with_feature("organizations:workflow-engine-notification-action")
```

Comment:
Since this is GA'd iirc, can we just clean this up first and remove these feature overrides now?